GCC 13+ no longer includes <cstdint> transitively, so uint64_t is undeclared.

diff --git a/external/VSRTL/interface/vsrtl_vcdfile.h b/external/VSRTL/interface/vsrtl_vcdfile.h
index 702db03..0f8a063 100644
--- a/external/VSRTL/interface/vsrtl_vcdfile.h
+++ b/external/VSRTL/interface/vsrtl_vcdfile.h
@@ -4,6 +4,7 @@
 #include <fstream>
 #include <functional>
 #include <iostream>
+#include <cstdint>
 #include <map>
 
 namespace vsrtl {
